Back-End/controllers/admin_controllers/instructors_controllers.js

```diff
@@ -1,6 +1,15 @@
 const db = require('../../db');
 
+const instructors_details = (req, res) => {
+    const query = `SELECT * FROM users WHERE role_id = 2`;
```

Comment:
This query needs to be dynamic, what if changes happen the role_id 2 is used for another role

Reply:
Point taken, but I'm referring to a specific id in a table. How can I make that dynamic?